### PR TITLE
About page: personal Github, Linkein link, as well as acknowledgments…

### PR DESCRIPTION
**Pull request by rmsepic**
I removed the pictures from the about page because they were landscape and profile pictures didn't look good in that form. 

I added our Github, Linkedin, and my resume, I can add anyone else's resume super easily if they like that feature.

I didn't add any description paragraphs because I thought it made it look a little too busy, and I wasn't quite sure what to put on there other than some type of corny bio. Instead, I wrote a light team bio, saying who we are.

I also removed Eric from the team, because it looked better with 4 then 5, but I added an acknowledgments section where we thank Eric, Xin, and Albara. 

Overall I was trying to keep the design minimalistic and clean, keeping the writing to a minimum. But, let me know what you think. 

--- Edit 10 May 2020 ---

I updated the homepage. I tried making a background that was a carousel of pointless packagings, but it did not work. The photos did not look good because they were not backgrounds, so I ended up deciding on a nice landscape because we have an environmental theme and we want a nice positive background. 

For examples of packaging, I put a carousel underneath the background. I like the idea of showing some examples, but it just does not look good as a background.

### CODE REVIEW

File: PP_APP/server/frontend/templates/about.html [hunk 1]
```diff
@@ -44,107 +44,109 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
...
                     <div class="d-flex h-100">
                         <div class="project-text w-100 my-auto text-center text-lg-left">
-                            <h4 class="text-white">Yoosuf Shafi</h4>
-                            <p class="mb-0 text-white">An example of where you can put an image of a project, or
-                                anything else, along with a description.</p>
+                            <h4 class="text-white">Kenny Li</h4>
+                            <p class="mb-0 text-white">
+                              <a class="about-link" target="_blank" href="https://github.com/kennylee9672">Github</a> |
+                              <a class="about-link" target="_blank" href="https://www.linkedin.com/feed/">LinkedIn</a>
+                            </p>
```
Comment (ShafiTek) [inline PP_APP/server/frontend/templates/about.html:88]: Please fix the Github and Linkedin links for both Kenny and I.

Comment (rmsepic) [reply PP_APP/server/frontend/templates/about.html]: My apologies, fixed

**Comment by ShafiTek** [comment]
Overall looks really good! And I like the fact that it is pretty minimalistic.

**Comment by ShafiTek** [comment]
The design starting to look pretty sweet. The footer looks neat!

File: PP_APP/server/frontend/templates/about.html [hunk 2]
```diff
@@ -44,107 +44,125 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
...
 
+        <h2 class="align-items-center text-center">&nbsp;</h2>
         <h2 class="align-items-center text-center">&nbsp;</h2>
         <h2 class="align-items-center text-center">&nbsp;</h2>
         <h1 class="align-items-center text-center">OUR TEAM</h1>
         <h2 class="align-items-center text-center">&nbsp;</h2>
 
+        <div class="featured-text text-center text-lg-left">
+            <p class="text-black-80">
+                Pointless Packaging is a project developed by senior UC Davis computer
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/templates/about.html:55]: I don't think the name of our project should be "Pointless Packaging", it sounds awkward. I'd just put the whole project name, "Eliminating Pointless Packaging Using Computer Vision", possibly italicize it too. That goes for all instances of the project name on the website.

File: PP_APP/server/frontend/templates/about.html [hunk 3]
```diff
@@ -44,107 +44,125 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
...
 
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+        <h1 class="align-items-center text-center">ACKNOWLEDGEMENTS</h1>
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+
+        <div class="featured-text text-center text-lg-left">
+            <p class="text-black-80">
+                Speical thanks to our client <a>Eric Law</a>
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/templates/about.html:155]: This is a great way to recognize the people who supported us!

**Review by fstamenkovic — CHANGES_REQUESTED** [review]
Great work on the front end. Couple of suggestions:
- I think our abstract for the Design Showcase should go under **Mission** on the About page. The current one is very similar but I think I saw a few grammar errors.
- I do not think that the second paragraph in the **Mission** section is necessary because we repeat that information later under **Our Team** page. I would delete it, and move the **Our Team** up right after the mission. If you don't think that moving team info up is a good idea, I still think the second paragraph should be removed because we shouldn't repeat the same info twice.
- In the **Our Team** part, I'd remove the part about demonstrating skills and instead replace it with the part that you have about exploring CV and machine learning while having a positive impact.

Great work and thanks!

**Review by kennylee9672 — APPROVED** [review]
Look good to me! Thanks!

File: PP_APP/server/frontend/templates/about.html (hunk 2 above)
Comment (rmsepic) [reply PP_APP/server/frontend/templates/about.html:55]: I italized it.

File: PP_APP/server/frontend/templates/about.html (hunk 3 above)
Comment (rmsepic) [reply PP_APP/server/frontend/templates/about.html:155]: Sweet. I removed the links I had for their names. After I thought about it, I figured we should ask their permission before doing something like adding their links.

**Review by fstamenkovic — APPROVED** [review]
I like the new look! We'll find some better images for the carousel before the end of the quarter.

Great work!

**Review by shafitek — APPROVED** [review]
Looking great!

**Pull request merged**